dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java

```diff
@@ -64,6 +64,12 @@
         "delete", "alter", "update", "create", "drop", "commit", "createdb",
         "createuser", "insert", "rename", "restore", "write" ).build();
 
+    public static final String CURRENT_USER_ID_VARIABLE = "_current_user_id";
+    public static final String CURRENT_USERNAME_VARIABLE = "_current_username";
+
+    public static final Set<String> BUILT_IN_VARIABLES = ImmutableSet.<String>builder().add(

```

Comment:
Rename to "standard variables" / `STANDARD_VARIABLES` ?

Reply:
Changed.